frontend/integration-tests/views/yaml.view.ts

```diff
@@ -1,16 +1,21 @@
 /* eslint-disable no-undef, no-unused-vars */
-
-import { $, $$, by, Key, browser, ExpectedConditions as until } from 'protractor';
+import { $, by, Key, browser, ExpectedConditions as until } from 'protractor';
+import { untilNoLoadersPresent } from './crud.view';
```

Comment:
Not sure how I feel about different views importing from each other.

Reply:
Maybe we should move that export into the protractor conf file